File: 58_sap_xep_xen_ke.cpp
#include<iostream>
#include<algorithm>
using namespace std ; 
int n ;
#define X 1000
int M[X] ; 
void solve() {
    int c = n-1 , d = 0;
    sort(M , M+n) ; 
    while(d<c){
        cout<<M[c]<<" "<<M[d]<<" "  ; d++  ; c-- ; 
    }
    if (d==c) cout<<M[c]<<" " ; 
    cout<<endl ; 
}

main() {
    int t; cin>>t  ; 
    while(t--) {
        cin>>n ; 
        for(int i=0 ; i<n ; i++) cin>>M[i] ; 
        solve() ; 
    }
}

File: 58_sap_xep_xen_ke.py
def solve():
    global M , n 
    M.sort() 
    d = 0 
    c = n-1 
    while d<c:
        print(M[c],M[d],end=" " )
        d+=1
        c-=1
    if d == c:
        print(M[d],end='')
    print() 


import math
t = int(input()) 
while t>0:
    n = int(input()) 
    M = list(map(int , input().split()))
    solve() 
    t-=1 